packages/code-studio/src/plugins/internal/ExampleAppPlugin.tsx

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable */
+// @ts-nocheck
```

Comment:
These files in `plugins/internal` should not be converted to tsx as we dynamically import them. Remove the `@ts-nocheck`